pkg/controller/deployment/deployment_controller_test.go

```diff
@@ -190,15 +195,25 @@ func (f *fixture) newController() (*DeploymentController, informers.SharedInform
 	return c, informers
 }
 
+func (f *fixture) runExpectError(deploymentName string) {
+	f.run_(deploymentName, true)
+}
+
 func (f *fixture) run(deploymentName string) {
+	f.run_(deploymentName, false)
+}
+
+func (f *fixture) run_(deploymentName string, expectError bool) {
```

Comment:
Go style does not use `_` for separating words, but sometimes it is used as a prefix or suffix to indicate "this is an internal implementation detail and you shouldn't usually access it directly". Since this is just a test, I didn't think it was worth trying to figure out a meaningful name to give `run_()`. I just needed to move it out of the way of the `run()` function that doesn't have the extra `bool`.